src/Component/Metadata/Section.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Resource\Metadata;
+
+#[\Attribute(\Attribute::TARGET_CLASS | \Attribute::IS_REPEATABLE)]
+final class Section
```

Comment:
Let's rename it to Routing
```suggestion
final class Routing
```

Calling it a `Section` implies it is a superior concept over the routing, while it is an optional configuration. What is more, I had issues understanding what it configures just by seeing naming. `Routing` seems to be much more readable and straightforward name to me. 

However, we should preserve `section` config, but I would recommend putting it as one of keys under `Routing` attribute